fr_CA/home/src/main/resources/rdf/tbox/firsttime/vitroAnnotations_fr_CA.ttl

```diff
@@ -0,0 +1,4427 @@
+# baseURI: http://vivoweb.org/ontology/vitroAnnotfr_CA
+
+@prefix : <http://vivoweb.org/ontology/vitroAnnotfr_CA#> .
```

Comment:
Are there functional changes in this file? The whitespace changes seem to be obscuring any such changes.